### ISSUE TITLE
Parsing time error 'GetMarketStatus'

### ISSUE THREAD

**Issue by andersc** [polygon-io/client-go#121]
When using the method **_GetMarketStatus_**

```go
res, err := gRestClient.GetMarketStatus(context.Background())
if err != nil {
   logger.Error(err)
}
```

I get this error ->

```

2022/05/17 14:58:11.343078 ERROR RESTY parsing time "2022-05-17T08:58:11-04:00" as "2006-01-02T15:04:05Z": cannot parse "-04:00" as "Z", Attempt 1
2022-05-17T14:58:14.200+0200    ERROR   failed to execute request: parsing time "2022-05-17T08:58:11-04:00" as "2006-01-02T15:04:05Z": cannot parse "-04:00" as "Z"
```

**Comment by morningvera** [comment]
thanks for catching this @andersc! i'll have a fix out for you soon

**Comment by morningvera** [comment]
This should be fixed in `v0.9.1`!

**Comment by morningvera** [comment]
Reopening, this is still not fixed
